modules/oc-mirror-oci-format.adoc

```diff
@@ -0,0 +1,130 @@
+// Module included in the following assemblies:
+//
+// * installing/disconnected_install/installing-mirroring-disconnected.adoc
+
+:_content-type: PROCEDURE
+[id="oc-mirror-oci-format_{context}"]
+= Mirroring Operator images in OCI format
+
+You can use the oc-mirror plug-in to mirror Operators in the Open Container Initiative (OCI) image format, instead of Docker v2 format. You can copy Operator images to a file-based catalog on disk in OCI format. Then you can then copy local OCI images to your target mirror registry.
+
+:FeatureName: Using the oc-mirror plug-in to mirror Operator images in OCI format
+include::snippets/technology-preview.adoc[]
+
+.Prerequisites
+
+* You have access to the internet to obtain the necessary container images.
+* You have installed the OpenShift CLI (`oc`).
+* You have installed the `oc-mirror` CLI plug-in.
+
+.Procedure
+
+. Create the image set configuration file to retrieve the catalogs and images that you require.
++
+.Example image set configuration file for copying to disk
+[source,yaml]
+----
+kind: ImageSetConfiguration
+apiVersion: mirror.openshift.io/v1alpha2
+mirror:
+ operators:
+ - catalog: registry.redhat.io/redhat/redhat-operator-index:v4.12
+   packages:
+   - name: aws-load-balancer-operator
+----
++
+[NOTE]
+====
+When using the OCI feature, only the `mirror.operators.catalog` setting is available for use.
+
+The `storageConfig` setting is ignored in favor of the location passed in to the `oc mirror` command.
+====
+
+. Run the `oc mirror` command to mirror the images from the specified image set configuration to disk:
++
+[source,terminal]
+----
+$ oc mirror --config=./imageset-config.yaml \ <1>
+  --use-oci-feature \                         <2>
+  --oci-feature-action=copy \                 <3>
+  oci://my-oci-catalog                        <4>
```

Comment:
Now we have bug for the relative path usage , we could use the full path here as example:
`oc-mirror --config config-operator.yaml  --use-oci-feature --oci-feature-action=copy  oci:///home/ocmirrortest/noo

Reply:
Thanks @zhouying7780! I'll update to make this a full path.

Is there a bug link for this that I could include as a known issue in the release notes (if we don't fix it by 4.12 GA)?

Reply:
CC @sherine-k @lmzuccarelli if you know the bug link for this? ^

Reply:
OCPBUGS-4516. We'll cherry-pick this for 4.12 as soon as it merges

Reply:
Thanks @sherine-k!

I'm assuming that I should remove the statement `This directory is created in the current working directory where the `oc mirror` command is run.` for now, since a full path has to be specified. Sound right?

We can always add it back when that bug is fixed.

Reply:
@sherine-k @zhouying7780 Updated to use a full path. Please re-review when you get a chance, thanks!

Preview: https://53150--docspreview.netlify.app/openshift-enterprise/latest/installing/disconnected_install/installing-mirroring-disconnected.html#oc-mirror-oci-format_installing-mirroring-disconnected